Dalamud/Interface/ImGuiNotification/Internal/ActiveNotification.cs

```diff
@@ -317,11 +268,6 @@ internal void RemoveNonDalamudInvocations()
         if (this.Icon is { } previousIcon && !IsOwnedByDalamud(previousIcon.GetType()))
```

Comment:
Change this line so that `IsOwnedByDalamud` check is also done for the IDTW stored inside `ForwardingSharedImmediateTexture`.